modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -65,10 +71,9 @@ spec:
 Because Loki requires log streams to be correctly ordered by timestamp, `labelKeys` always includes the `kubernetes_host` label set, even if you do not specify it. This inclusion ensures that each stream originates from a single host, which prevents timestamps from becoming disordered due to clock differences on different hosts.
 ====
 
-
-. Create the CR object:
+. Apply the `ClusterLogForwarder` CR object:
```

Comment:
Per the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command), suggestion:

```suggestion
. Apply the `ClusterLogForwarder` CR object by running the following command:
```